src/corelocation.cs

```diff
@@ -428,13 +446,25 @@ partial interface CLLocationManagerDelegate
 
 #if !MONOMAC
 		[NoWatch][NoTV]
+		[Deprecated (PlatformName.iOS, 13,0, message: "Use 'DidRangeBeaconsSatisfyingConstraint' instead.")]
 		[iOS (7,0), Export ("locationManager:didRangeBeacons:inRegion:"), EventArgs ("CLRegionBeaconsRanged")]
 		void DidRangeBeacons (CLLocationManager manager, CLBeacon [] beacons, CLBeaconRegion region);
 
+		[NoWatch, NoTV, NoMac, iOS (13,0)]
+		[Export ("locationManager:didRangeBeacons:satisfyingConstraint:")]
+		[EventArgs ("CLRegionBeaconsConstraintRanged")]
+		void DidRangeBeaconsSatisfyingConstraint (CLLocationManager manager, CLBeacon[] beacons, CLBeaconIdentityConstraint beaconConstraint);
```

Comment:
Can `DidRangeBeaconsSatisfyingConstraint` be shortened to `DidRangeBeacons` - since the params are different it won't cause a namespace issue, right?

Reply:
Good catch! It was my first attempt too - but sadly it can't :( 

The problem are that:
* the name is already used in the type (even if overloads are fine most of the time); 
* a `*Delegate` type;
* it uses  `[EventArgs]`;

so this cause a conflict in the generated code.

I don't have this PR built right now - but I'll update the comment with the code (easier to see what's happening this way).